src/ctype/CTypeSchema.ts

```diff
@@ -133,7 +133,46 @@ export const CTypeWrapperModel = {
       type: 'string',
       minLength: 1,
     },
-    metamodel: {
+  },
+  required: ['schema'],
+}
+
+export const CTypeWrapperMetadata = {
+  metadata: {
+    $id: 'http://kilt-protocol.org/draft-01/ctype-metadata',
+    $schema: 'http://json-schema.org/draft-07/schema#',
+    title: {
+      type: ' string',
```

Comment:
As far as I remember, we always have at least a `default` prop here and are expecting more props for other languages

Reply:
I changed it to default again.